### DISCUSSION TITLE
Are snapd and lxd mandatory on Ubuntu ?

### DISCUSSION THREAD

**Discussion by afbjorklund** [lima-vm/lima#758] [Q&A] [answered]
Or could they be disabled by default, and opt-in later if they are desired ?

> LXD is installed in the default Ubuntu template, so there is no `lxd.yaml`

Note: lxd is installed by snap, but nothing else from what I can tell

```
Name    Version       Rev    Tracking         Publisher   Notes
core20  20220114      1328   latest/stable    canonical✓  base,disabled
core20  20220304      1376   latest/stable    canonical✓  base
lxd     4.22          22292  latest/stable/…  canonical✓  disabled
lxd     4.24-c92c0b2  22754  latest/stable/…  canonical✓  -
snapd   2.54.4        15177  latest/stable    canonical✓  snapd
```

**Comment by AkihiroSuda** [comment, marked as answer]
They are installed by default (not our choice), but you can uninstall them manually

**Reply by afbjorklund** [reply]
They are installed by default in "ubuntu.yaml", but I guess there _is_ a choice for "default.yaml" if it is not bound to upstream Ubuntu* ?

\* i.e. similar to alpine-lima vs alpine

Will leave it for now, it is only some extra packages and services - was mostly curious about the extra mounts in the systemd output...
